plugins/modules/fusion_api_client.py

```diff
@@ -31,6 +31,11 @@
     default: present
     choices: [ present, absent ]
     type: str
+  display_name:
```

Comment:
display name is the same as name. It should be an alias and conflict value with name.

Reply:
Can you add more context, please?

Reply:
You're right, but I think this is actually backend bug, because general model tracks name and display name separately and create model has only display name. This should be resolved.

Reply:
Wait it's actually OpenAPI spec that has it wrong, not backend.

Reply:
So in essence display_name is a valid field, name does not exist.

Reply:
Makes sense now. Thanks for clarification. Will close this PR.

Reply:
Thanks for noticing @DenDen039 @jankodera-ps